src/Form/Ingest/Review.php

```diff
@@ -146,13 +146,19 @@ public function submitActivation(array &$form, FormStateInterface $form_state) {
    *   A reference to the batch context.
    */
   public function runBatchOp(MigrationInterface $migration, MigrateExecutable $e, &$context) {
+    assert($e instanceof MigrateBatchExecutable);
```

Comment:
Right, this was rolled in the separate executable for whatever reason...

```suggestion
    assert($e instanceof MigrationRollbackBatch);
```